### DISCUSSION TITLE
v0.9.0-test

### DISCUSSION THREAD

**Discussion by timothyschoen** [plugdata-team/plugdata#1544] [General]
Pre-release version of plugdata v0.9.0

Headlines:
- Huge performance improvements because of GPU accelerated rendering!
- Ships with Gem built in. Gem objects need the "Gem/" prefix, unless you add Gem to the list of libraries to load.
- Patches load a lot faster now
- Latest ELSE master branch changes, including updated MERDA modules
- Latest pure-data master branch changes
- Improved compatibility with [drawcurve], [fillcurve] and [plot] drawing
- Redesigned GraphOnParent area resizer
- Plugin version now displays the current latency in the statusbar if it has been changed from the default
- Improvements to pd-vanilla compatibility, patches should look even more similar to their vanilla counterparts than before
- Improved Heavy toolchain download speed
- Fixed [keyboard] bugs
- Many bugfixes!

Breaking changes:
- [cyclone/scope~] and [else/oscope~] have been replaced with [else/scope~]. These two were essentially the same object anyway.

<hr /><em>This discussion was created from the release <a href='https://github.com/plugdata-team/plugdata/releases/tag/v0.9.0-test'>v0.9.0-test</a>.</em>

**Comment by AlexMitchellMus** [comment]
* New connection activity display
* Connection arrows drawn between segments when connection is segmented
* Windows build now uses system rounded window border and dropshadow. (Straight for w10)
* Connections can be set behind / in front for both edit & locked mode.

**Comment by dromer** [comment]
I hope there will be some more fixes to the compositing since a lot of parts are either unusable or super weird/quirky in non-compositing environments.

**Reply by timothyschoen** [reply]
There are some things I can try to improve (like the callout boxes and object DnD), but also things where not having compositing means we don't have the tools we need to make stuff look good.

For example, the dialog backgrounds can't be transparent, since we need to place a window on top of the openGL rendered content. Without compositing, it's not possible to fix that.

**Reply by dromer** [reply]
Even with compositing it works super awkward. since it covers everything in the screen (see: https://github.com/plugdata-team/plugdata/issues/1548).

I really feel there are better ways to deal with this, but I don't have enough knowledge/experience to suggest anything good :#
